setup.py

```diff
@@ -17,7 +17,7 @@ def run_tests(self):
 pypi_name = "nisystemlink-clients"
 
 packages = find_namespace_packages(include=["systemlink.*"]) + find_packages(
-    exclude=["systemlink", "examples", "tests"]
+    exclude=["systemlink", "systemlink.*", "examples", "examples.*", "tests", "tests.*"]
```

Comment:
I might be misunderstanding what this is doing, but what is there besides these? Is the `find_packages` call actually returning anything that `find_namespace_packages` isn't?

Reply:
The intent was to have this statement be a "catch-all" for all normal packages, i.e. the ones which don't have special behavior due to namespace package rules. It semantically matches the previous implementation where all valid packages in the repo are included except for a select set of directories.

Indeed, it is redundant in the current package layout, though. Might as well remove it.